lib/src/ui/pages/presentation/presentation_steps_context_provider.dart

```diff
@@ -32,7 +32,7 @@ extension PresentationImageTypeExtenstions on PresentationImageType {
   Widget getImage() {
     switch (this) {
       case PresentationImageType.flutterLogo:
-        return const FlutterLogo(colors: Colors.grey);
+        return const FlutterLogo();
```

Comment:
od czasu Twojego pusha bitrise się wywala, powodem było to, że używasz starej bety lub kanału stable lokalnie, na bitrise, tak jak się kiedyś umawialiśmy używamy bety, która jest na bieżąco aktualizowana. No i w najnowszej becie nie da się w ten sposób zmienić koloru loga